Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM alpine:3.15
+WORKDIR /home
+RUN apk add --no-cache openssh
+RUN apk add --no-cache expect
+CMD [ "sleep", "infinity" ]
```

Comment:
Let's remove the `CMD` which will make it use the base image's CMD. Or keep it as `CMD ["/bin/sh"]`